src/compute/src/logging/compute.rs

```diff
@@ -609,6 +687,47 @@ impl<A: Allocate> DemuxHandler<'_, '_, A> {
         }
     }
 
+    fn handle_error_count(&mut self, export_id: GlobalId, diff: i64) {
+        if let Some(export) = self.state.exports.get_mut(&export_id) {
+            export.error_count += diff;
+
+            let ts = self.ts();
+            self.output.error_count.give((export_id, ts, diff));
+        };
+
+        // Send the diff to any known error forwards.
+        // Note that we have to do this even if this export has already been dropped, as its
+        // dependants might still be alive.
+        if let Some(forwards) = self.state.error_forwards.get(&export_id) {
+            for id in forwards.clone() {
+                self.handle_error_count(id, diff);
+            }
+        }
+    }
+
+    fn handle_error_forward(&mut self, from_export: GlobalId, to_export: GlobalId) {
+        // Log the initial error count.
+        if let Some(export) = self.state.exports.get(&from_export) {
+            self.handle_error_count(to_export, export.error_count);
+        } else {
+            error!(
```

Comment:
Possible to cover?

Reply:
This part has been removed. Wouldn't have been possible to cover because this was basically a "gentle assert" that would have alerted us to a bug without crashing the whole progress.